### DISCUSSION TITLE
Icon for Anchor Links

### DISCUSSION THREAD

**Discussion by j-steinbach** [srid/emanote#137] [Ideas]
Add an icon (often a [chain](https://fontawesome.com/v5.15/icons?d=gallery&p=2&q=link%20chain)) to sections/[anchor links](https://www.w3docs.com/snippets/html/how-to-create-an-anchor-link-to-jump-to-a-specific-part-of-a-page.html)

Examples:

- https://fomantic-ui.com/collections/menu.html#labeled-icon
- https://borgbackup.readthedocs.io/en/stable/#main-features

Observe how there is a small icon on either the left or the right of the section heading, and clicking it navigates to the section and updates the url.

Also try [this](https://note.ema.srid.ca/demo/embed#images) and [this](https://note.ema.srid.ca/demo/embed#files) and observe how there is no link icon.

**Comment by kukimik** [comment]
The main problem with this is that the `<base>` tag breaks the anchor links (see the hack in  https://github.com/EmaApps/emanote/blob/b1e871130d2d65eea04d9b67e059b8bcca4ac79d/default/templates/components/pandoc.tpl#L52). So we would have to put the full path to the current note in the href. I'm not sure how to do it. Apart from this it looks easy to do.

**Reply by kukimik** [reply]
After #361 this is no longer a problem, we can use `${ema:note:url}` to get the absolute url.

**Comment by srid** [comment]
https://github.com/srid/emanote/issues/480
